pkg/storaged/client.js

```diff
@@ -854,29 +854,42 @@ function init_model(callback) {
                 });
     }
 
-    function enable_udisks_features() {
+    async function enable_udisks_features() {
         if (!client.manager.valid)
             return Promise.resolve();
-        if (!client.manager.EnableModules)
+        if (!client.manager.EnableModule)
```

Comment:
This check seems a bit strange, not sure what we guard against? Shall we just drop it?